src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/resource/PrisonScheduleResourceIntegrationTest.kt

```diff
@@ -0,0 +1,173 @@
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.MediaType
+import org.springframework.test.web.reactive.server.WebTestClient
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.ScheduleItem
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.MOORLAND
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.WERRINGTON
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.courtBookingRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.hasSize
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.probationBookingRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.helper.werringtonLocation
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.AppointmentType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.BookingType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.CreateVideoBookingRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.ProbationMeetingType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.VideoBookingRepository
+import java.time.LocalDate
+import java.time.LocalTime
+
+class PrisonScheduleResourceIntegrationTest : IntegrationTestBase() {
+  @Autowired
+  private lateinit var videoBookingRepository: VideoBookingRepository
+
+  @Test
+  fun `should return no items on the schedule with no bookings present`() {
+    videoBookingRepository.findAll() hasSize 0
+
+    val scheduleResponse = webTestClient.getPrisonSchedule(
+      username = "PRISON-USER",
+      prisonCode = MOORLAND,
+      date = LocalDate.now(),
+    )
+
+    assertThat(scheduleResponse).hasSize(0)
+  }
+
+  @Test
+  fun `should return one scheduled item when a court booking is made with one appointment`() {
+    val bookingCreator = "BOOKING_CREATOR"
+
+    videoBookingRepository.findAll() hasSize 0
+
+    prisonSearchApi().stubGetPrisoner("A1111AA", WERRINGTON)
+    locationsInsidePrisonApi().stubPostLocationByKeys(setOf(werringtonLocation.key), WERRINGTON)
+
+    // Will default to tomorrow's date
+    val courtBookingRequest = courtBookingRequest(
+      courtCode = "DRBYMC",
+      prisonerNumber = "A1111AA",
+      prisonCode = WERRINGTON,
+      location = werringtonLocation,
+      startTime = LocalTime.of(12, 0),
+      endTime = LocalTime.of(12, 30),
+      comments = "integration test court booking comments",
+    )
+
+    webTestClient.createBooking(bookingCreator, courtBookingRequest)
+
+    // Check for tomorrow's date
+    val scheduleResponse = webTestClient.getPrisonSchedule(
+      username = "PRISON-USER",
+      prisonCode = WERRINGTON,
+      date = LocalDate.now().plusDays(1),
```

Comment:
FYI there is a helper function `tomorrow()` which you can use here, same goes for `today()` and `yesterday()`.

Reply:
Now uses it.